Example/EnvChanger_ExampleUITests/EnvChanger_ExampleUITests.swift

```diff
@@ -0,0 +1,69 @@
+//
+//  EnvChanger_ExampleUITests.swift
+//  EnvChanger_ExampleUITests
+//
+//  Created by Nikola B Nikolov on 29.11.21.
+//  Copyright © 2021 CocoaPods. All rights reserved.
+//
+
+import XCTest
+
+class EnvChanger_ExampleUITests: XCTestCase {
+    
+    // MARK: - Properties
+    var app: XCUIApplication!
+    var envButton: XCUIElement { app.buttons["ENV"] }
+    var environmentChangedAlert: XCUIElement { app.alerts["Environment successfully changed."] }
+    var closeEnvChangedAlertButton: XCUIElement { environmentChangedAlert.scrollViews.otherElements.buttons["Great!"] }
+    
+    // MARK: - Setup methods
+    override func setUpWithError() throws {
+        try super.setUpWithError()
+        continueAfterFailure = false
+        app = XCUIApplication()
+        app.launch()
+    }
+    
+    override func tearDownWithError() throws {
+        app = nil
+    }
+    
+    // MARK: - Testing methods
+    func testEnvButtonShowsEnvironmentOptions() {
+        // given
+        let selectProductionButton = app.sheets.firstMatch.scrollViews.otherElements.buttons["https://production.server.com/"]
```

Comment:
[optional]

Could this be retrieved from some constants, maybe from the project itself if possible?